src/Components/Config/configHook.tsx

```diff
@@ -1,14 +1,34 @@
 import { useEffect, useState } from 'react';
 import { configEndpoint, header } from '../../apiCalls';
-
 import { ConfigApiResponse } from '../../Types/Config';
 import { Config } from '../../Types/Config';
+import { FormattedMessage } from 'react-intl';
+
+// Recursive function
+function transformItem(item: any): any {
+  if (typeof item !== 'object' || item === null) {
+    return item;
+  }
+
+  if (item.hasOwnProperty('_label')) {
+    return <FormattedMessage id={item._label} defaultMessage={item._default_message} />;
+  }
+
+  const config: Config = {};
+  for (const key in item) {
+    if (item.hasOwnProperty(key)) {
+      config[key] = transformItem(item[key]);
+    }
+  }
+  return config;
+}
 
+// Main function with recursion
```

Comment:
I don't think that this comment is very helpful. What do you think about removing it.